.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: main
+
+on:
+  pull_request:
+  push:
+    branches:
+    - main
+ 
+
+jobs:
+  build:
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Use OCaml ${{ matrix.ocaml-compiler }}
+        uses: ocaml/setup-ocaml@v2
+        with:
+          opam-pin: false
+          opam-depext: false
+          ocaml-compiler: ocaml.4.13.1,ocaml-option-mingw
```

Comment:
We should run the latest version of OCaml - 5.1.0.

Reply:
Thank you so much. I'm really glad to hear that because it's my first time implementing continuous integration. I'd be sure to make those changes ASAP. Also, the version on my system is 4.13.1 and it says it's the latest version. Do I have to upgrade to 5.0.0 ?